src/components/PostEditor/PostEditor.js

```diff
@@ -22,13 +22,18 @@ import DatePicker from 'components/DatePicker'
 import UploadAttachmentButton from 'components/UploadAttachmentButton'
 import SendAnnouncementModal from 'components/SendAnnouncementModal'
 import PublicToggle from 'components/PublicToggle'
+import AnonymousVoteToggle from './AnonymousVoteToggle/AnonymousVoteToggle'
+import SliderInput from 'components/SliderInput/SliderInput'
+import Dropdown from 'components/Dropdown/Dropdown'
 import styles from './PostEditor.scss'
 import { PROJECT_CONTRIBUTIONS } from 'config/featureFlags'
 import { MAX_POST_TOPICS } from 'util/constants'
+import { setQuerystringParam } from 'util/navigation'
 import { sanitizeURL } from 'util/url'
 import Tooltip from 'components/Tooltip'
-import { setQuerystringParam } from 'util/navigation'
 
+
+const emojiOptions = ['', '1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣', '🔟', '✅✅', '👍', '👎', '⁉️', '‼️', '❓', '❗', '🚫', '➡️', '🛑', '✅', '🛑🛑', '🌈', '🔴', '🔵', '🟤', '🟣', '🟢', '🟡', '🟠', '⚫', '⚪', '🤷🤷', '📆', '🤔', '❤️', '👏', '🎉', '🔥', '🤣', '😢', '😡', '🤷', '💃🕺', '⛔', '🙏', '👀', '🙌', '💯', '🔗', '🚀', '💃', '🕺', '💯',]
```

Comment:
why limit emoji options? what if people want to choose between an Italian restaurant and a Chinese restaurant?

Reply:
1) In the balance between max flexibility and channeling users to the most useful/sensible options, channeling makes more sense for proposals.

2) Can't do double emojis with the regular emoji picker. (Although I note you aren't a fan of double emojis but they are a clear way to handle gradients of agreement)

Reply:
i still disagree here about this, i guess we could bring to the group. also 100 is listed twice :) and does the order matter? if you are going to have double emojis then i think the single version should go right before the double one.